sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 7.9.0b2 (Unreleased)
+## 7.10.0b1 (2023-04-11)
```

Comment:
Won't this be b2 release?

Reply:
not unless we release the beta second, but I think we're releasing the GA first? and that will be 7.9.0, so this will be the next beta I think.